File: 125-valid-palindrome/valid_palindrome.dart
class Solution {
  bool isPalindrome(String s) {
    s = s.toLowerCase();
    s = s.replaceAll(RegExp(r'[^a-z0-9]'), '');
    return s == s.split('').reversed.join('');
  }
}

void main() {
  final solution = Solution();
  print(solution.isPalindrome('arara'));
}

File: 125-valid-palindrome/valid_palindrome.py
class Solution(object):
    def is_palindrome(self, s):
        s = ''.join(e for e in s if e.isalnum()).lower()
        return s == s[::-1]


if __name__ == "__main__":
    s = Solution()
    print(s.is_palindrome("arara"))